web/styling/positioning-tips/position-elements-with-position-sticky.md

```diff
@@ -43,3 +42,10 @@ Which of the following browsers *does not* support `position: sticky`? ???
 - Firefox
 - Safari
 - iOS Safari
+
+---
+## Footnotes
+
+[1:position fixed]
+
+The `position: fixed` element positions relative to the viewport. This means it will always stay in the same place even if you scroll up or down the page.
```

Comment:
I don't think `element` is correct here. afaik `position` is the property, `fixed` is the value, and `position: fixed` is the declaration

```suggestion
The `position: fixed` declaration positions relative to the viewport. This means it will always stay in the same place even if you scroll up or down the page.
```